doc/classes/@GDScript.xml

```diff
@@ -673,27 +760,48 @@
 			</return>
 			<description>
 				Return a random floating point value between 0 and 1.
+				[codeblock]
+				# prints 0.375671
+				print(randf())
+				[/codeblock]
 			</description>
 		</method>
 		<method name="randi">
 			<return type="int">
 			</return>
 			<description>
-				Return a random 32 bits integer value. To obtain a random value between 0 to N (where N is smaller than 2^32 - 1), you can use remainder. For example, to get a random integer between 0 and 19 inclusive, you can use randi() % 20.
+				Return a random 32 bits integer value. To obtain a random value between 0 to N (where N is smaller than 2^32 - 1), you can use remainder.
```

Comment:
Shorter: 
`Return a random 32 bits integer. Use remainder to obtain a random value between 0 to N (where N is smaller than 2^32 - 1).`

Reply:
Or more accurate:
`Return a random 32 bits positive integer (comprised between 0 and 2^32 - 1). Use the remainder operator to restrict the value to a [0;N[ range.` (or `to a given range` if people would freak out seeing `[0;N[` ;))